src/Mafin.Web.Api.Rest/MafinHttpClient.cs

```diff
@@ -280,4 +280,15 @@ public MafinHttpClient(HttpMessageHandler handler, bool disposeHandler)
 #pragma warning restore CA1054
 
     private static CancellationToken ResolveToken(CancellationToken? token) => token ?? CancellationToken.None;
+
+    // Added for interoperability between targets
+    private async Task<HttpResponseMessage> GetCorrectPatchImplementationResponse(Uri? requestUri, HttpContent? content, CancellationToken cancellationToken)
+    {
+#if NETSTANDARD2_1_OR_GREATER
+        var response = await PatchAsync(requestUri, content, cancellationToken).ConfigureAwait(false);
+#else
+        var response = await SendAsync(new HttpRequestMessage(new HttpMethod("PATCH"), requestUri) { Content = content }, cancellationToken).ConfigureAwait(false);
```

Comment:
`HttpRequestMessage` instance is not disposed.

Reply:
@nvborisenko could you please exapnd what you propose here?
I believe using simple `using` statement would just dispose the object prematurely as it should be passed out of the scope we control.
Please, correct me if I'm wrong here.

Reply:
```csharp
using var reqMessage = new HttpRequestMessage(new HttpMethod("PATCH");
var response = await SendAsync(reqMessage, requestUri) { Content = content }, cancellationToken).ConfigureAwait(false);
```

Reply:
Seems you haven't read my comment to end, as I expressed concerns using this specific way you're proposing.
Anyways, I'd test it first and return back with the results.

Reply:
@nvborisenko,
I've written a short test, and as I predicted your proposed change triggers the following exception:
```
System.Net.Http.HttpRequestException : An error occurred while sending the request.
---- System.ObjectDisposedException : Cannot access a disposed object.
Object name: 'System.Net.Http.StringContent'.
```
The code without disposing `HttpRequestMessage` instance works as expected.
I also checked `HttpClient` code in microsoft repository - seems they are not disposing the message in the client either.

Reply:
This is because `HttpContent` came from outside.

Reply:
You're right, thanks for helping figuring out the cause!
TIL :)